simpleflow/execute.py

```diff
@@ -1,14 +1,19 @@
+import re
 import sys
 import subprocess
 import functools
 import json
 import cPickle as pickle
 import base64
+import logging
 
 
 __all__ = ['program', 'python']
 
 
+logger = logging.getLogger('simpleflow')
```

Comment:
Let me suggest `logger = logging.getLogger(__name__)` (supports renaming of the module and is our usual convention).